packages/aila/src/core/chat/AilaStreamHandler.ts

```diff
@@ -212,6 +216,41 @@ export class AilaStreamHandler {
           return quiz;
         },
         fetchRagData: async ({ document }) => {
+          const { subject, keyStage, topic, title } = document;
+          invariant(title, "Document title is required to fetch RAG data");
+
+          const isMaths = subject?.toLowerCase().startsWith("math") ?? false;
+
+          if (!isMaths) {
+            // temporaryily use only RAG if subject is not maths -- we only have maths in the new RAG
+            const relevantLessonPlans = await fetchRagContent({
+              title,
+              subject,
+              topic: topic ?? undefined,
+              keyStage,
+              id: this._chat.id,
+              k:
+                this._chat.aila?.options.numberOfRecordsInRag ??
+                DEFAULT_NUMBER_OF_RECORDS_IN_RAG,
+              prisma: globalPrisma,
+              chatId: this._chat.id,
+              userId: this._chat.userId,
+            });
+
+            return relevantLessonPlans.map((lessonPlan) => ({
+              ...lessonPlan,
+              starterQuiz: {
+                version: "v2",
+                questions: [],
+                imageAttributions: [],
+              },
+              exitQuiz: {
+                version: "v2",
+                questions: [],
+                imageAttributions: [],
+              },
```

Comment:
I know this is WIP, but it might be worth a comment to explain why we don't need quizzes in the rag